packages/components-js/tests/e2e/puppeteer/specs/scroller.e2e.ts

```diff
@@ -24,19 +24,23 @@ type InitOptions = {
   otherMarkup?: string;
   tag?: 'a' | 'button' | 'span';
   scrollToPosition?: ScrollToPosition;
+  scrollbar?: boolean;
 };
 
 const initScroller = (opts?: InitOptions) => {
-  const { amount = 8, isWrapped, otherMarkup = '', tag = 'button', scrollToPosition } = opts || {};
+  const { amount = 8, isWrapped, otherMarkup = '', tag = 'button', scrollToPosition, scrollbar } = opts || {};
 
   const elementAttributes = tag === 'a' ? ' onclick="return false" href="#"' : '';
   const elements = Array.from(Array(amount))
     .map((_, i) => `<${tag}${elementAttributes}>Button ${i + 1}</${tag}>`)
     .join('');
 
-  const attrs = scrollToPosition ? `scroll-to-position="{ scrollPosition: ${scrollToPosition.scrollPosition} }"` : '';
+  const attrs = [
+    scrollToPosition ? `scroll-to-position="{ scrollPosition: ${scrollToPosition.scrollPosition} }"` : '',
+    scrollbar ? `scrollbar="${scrollbar}"` : '',
+  ];
 
-  const content = `<p-scroller ${attrs}>
+  const content = `<p-scroller ${attrs.join(' ')}>
```

Comment:
Please do the join directly where the array is defined to be consistent with other tests.

Reply:
Fixed it